CONTRIBUTING.md

```diff
@@ -34,11 +36,21 @@ $ git clone https://github.com/koordinates/kart.git
 $ cd kart
 $ git submodule update --init --recursive
 ```
+
+### Setting Python3_ROOT
+
+CMake will try to automatically find a Python3 install on your system with which to build Kart.
+However, to avoid any issues caused by different Python implementations, it is recommended to force CMake
+to use Python 3.10, by supplying the flag -DPython3_ROOT - for example:
+`-DPython3_ROOT=$(realpath $(which python3.10) | sed 's@/bin/.*@/@')`
```

Comment:
can also do `-DPython3_EXECUTABLE=$(command -v python3.10)` — dunno which is better? Executable might be easier for users.

Reply:
Fine by me as long as this is as powerful as setting DPython3_ROOT, ie, cmake correctly finds the root from the executable and doesn't require you to set any other DPython3 variables

Reply:
Seems to work, done